ci-operator/config/openshift-service-mesh/sail-operator/openshift-service-mesh-sail-operator-main.yaml

```diff
@@ -162,15 +163,15 @@ tests:
           memory: 200Mi
       timeout: 2h0m0s
     workflow: servicemesh-istio-e2e-hypershift
-- as: e2e-ocp-4-17
+- as: e2e-ocp-4-18
   optional: true
   steps:
     cluster_profile: ossm-aws
     dependencies:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:next
```

Comment:
this `OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE` is a right way to use 4.18 builder image. I will add comment in the above section.